### DISCUSSION TITLE
Uğurlu sayıları bulan program

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus.github.io#67] [Soru Cevap]
Ayşegül, Buket, Cansu , Derya adlı dört arkadaşın her biri birinden farklı 4 uğurlu sayısı vardır. Bu sayıların toplamı 15, ve her uğurlu sayıya üyelerin baş harfini verirsek; A ve C nin toplamı D den küçüktür. B ve C nin toplamı A dan küçüktür. B, C den küçüktür. Sayılar pozitif ve birbirinden farklıdır. Uğurlu sayıları bulan program

**Comment by DeniZZece** [comment]
```
#include<stdio.h>
int main()
{
	int a, b, c, d;
	for (a=1;a<15;a++)
	{
		for (b=1;b<15;b++)
		{
			for (c=1;c<15;c++)
			{
				for (d=1;d<15;d++)
				{
					if (a+b+c+d==15 && a+c<d && b+c<a && b<c)
					{
						printf("%d %d %d %d",a,b,c,d);
						break;
					}
				}
			}
		}
	}
}

```
